blocks/page_list/templates/itinerary/Walk.jsx

```diff
@@ -0,0 +1,35 @@
+import { dateFormatted } from './Itinerary-Utils';
+import React from 'react';
+
+
+const Walk = ({title, start, meeting, remove, key}) => {
+  return (
+    <div>
+      <div className="itinerary-walk clearfix">
```

Comment:
Never use clearfix. It's semantically meaningless. Use the `.clearfix()` mixin inside the less selector for this element instead.